File: Examenes/EX2/22-1/BackEnd_CSharp/ConferenceSoftNETRemoting/ConferenceSoftModel/Integrante.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConferenceSoftModel
{
    public class Integrante
    {
        private int _idIntegrante;
        private TipoIntegrante _tipoIntegrante;
        private String _codigoPUCP;
        private String _nombre;
        private String _apellidoPaterno;
        private bool _activo;

        public int IdIntegrante { get => _idIntegrante; set => _idIntegrante = value; }
        public TipoIntegrante TipoIntegrante { get => _tipoIntegrante; set => _tipoIntegrante = value; }
        public string CodigoPUCP { get => _codigoPUCP; set => _codigoPUCP = value; }
        public string Nombre { get => _nombre; set => _nombre = value; }
        public string ApellidoPaterno { get => _apellidoPaterno; set => _apellidoPaterno = value; }
        public bool Activo { get => _activo; set => _activo = value; }
    }
}

File: Examenes/EX2/22-1/BackEnd_JAVA/ConferenceSoftModel/src/pe/edu/pucp/confesoft/model/Integrante.java
package pe.edu.pucp.confesoft.model;

import java.io.Serializable;

public class Integrante implements Serializable{
    private int idIntegrante;
    private TipoIntegrante tipoIntegrante;
    private String codigoPUCP;
    private String nombre;
    private String apellidoPaterno;
    private boolean activo;

    public int getIdIntegrante() {
        return idIntegrante;
    }

    public void setIdIntegrante(int idIntegrante) {
        this.idIntegrante = idIntegrante;
    }

    public TipoIntegrante getTipoIntegrante() {
        return tipoIntegrante;
    }

    public void setTipoIntegrante(TipoIntegrante tipoIntegrante) {
        this.tipoIntegrante = tipoIntegrante;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellidoPaterno() {
        return apellidoPaterno;
    }

    public void setApellidoPaterno(String apellidoPaterno) {
        this.apellidoPaterno = apellidoPaterno;
    }

    public boolean isActivo() {
        return activo;
    }

    public void setActivo(boolean activo) {
        this.activo = activo;
    }

    public String getCodigoPUCP() {
        return codigoPUCP;
    }

    public void setCodigoPUCP(String codigoPUCP) {
        this.codigoPUCP = codigoPUCP;
    }
    
}